schema/herocard-response-schema.json

```diff
@@ -38,6 +39,15 @@
       "required": ["href"]
     },
 
+    "openInLink": {
```

Comment:
Added a new definition instead of using the existing "link". So that now i can enforce both 'text' and 'href' are required.